src/Tool/Commands/SqlServerCommand.cs

```diff
@@ -257,16 +252,7 @@ async Task GetMinimums(CancellationToken cancellationToken)
                         {
                             foreach (var table in db.Tables.Where(t => t.MinRowVersion is null))
                             {
-                                using (var cmd = conn.CreateCommand())
-                                {
-                                    cmd.CommandText = $"select min(RowVersion) from {table.FullName} with (nolock);";
-                                    var value = await cmd.ExecuteScalarAsync(cancellationToken);
-
-                                    if (value is long rowversion)
-                                    {
-                                        table.MinRowVersion = rowversion;
-                                    }
-                                }
+                                table.MinRowVersion = await table.GetMaxRowVersion(conn, cancellationToken);
```

Comment:
Although I understand what this code does today. I'm not sure I'll remember it 6 months from now. Maybe we should rename MinRowVersion to something else that suggests it's the MaxRowVersion the first time it was sampled.